kalite/control_panel/templates/control_panel/partials/_device_table.html

```diff
@@ -63,6 +63,7 @@
                                     {% endif %}
                                 </td>
                                 <td>
+                                    <div style="margin-top: -24px;">
```

Comment:
We should avoid inline styling like this. Can you put this in a stylesheet somewhere?